gcloud/bigquery/client.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""gcloud bigquery client for interacting with API."""
+"""Client for interacting with the Google Cloud BigQuery API."""
```

Comment:
I think this is a weird one, like Compute Engine or App Engine, doesn't come prefixed with Cloud.